src/device-manager/java/src/nl/Weave/DataManagement/WdmClientImpl.java

```diff
@@ -200,9 +214,12 @@ private CompletionHandler requireCompletionHandler() {
 
     private native void init();
     private native void setNodeId(long wdmClientPtr, long nodeId);
+    private native void setEventFetchingTimeout(long wdmClientPtr, long timeout);
```

Comment:
Can this be removed now that the timeout parameter is part of the fetch API?

Reply:
yep, it should be removed.

Reply:
This function is mistakenly not removed, removed in latest commits.